fe/fe-core/src/main/java/com/starrocks/catalog/CatalogUtils.java

```diff
@@ -235,4 +236,16 @@ public static int calBucketNumAccordingToBackends() {
         }
         return bucketNum;
     }
+
+    public static void maskProperties(Map<String, String> properties) {
+        // aws.s3.access_key -> ******
+        properties.computeIfPresent(CloudConfigurationConstants.AWS_S3_ACCESS_KEY, (k, v) -> "******");
```

Comment:
Extract a common function to mask, like `maskCloudCredential()` , we will need to mask like azure, gcp in the future.

Reply:
i think maskCloudCredential can use to mask all ,just to add:
properties.computeIfPresent(CloudConfigurationConstants.AZURE_ACCESS_KEY, (k, v) -> "******");